### ISSUE TITLE
Point users to the online help

### ISSUE THREAD

**Issue by bochecha** [glimpse-editor/Glimpse#145]
Since #81 we have an Appdata file showing off Glimpse in software centers.

One thing these will show is a link to the online help, if the project has one.

This is done with [the `<url type="help">` tag](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url).

Currently it is commented out, but once we deploy a help site somewhere, we should uncomment it and set the right URL.

**Comment by TrechNex** [comment]
The internal help menu points users to the upstream help currently. We may have to push this out to a later 0.x release.

**Comment by bochecha** [comment]
> Addendum: Our own online help should eventually be https://wiki.glimpse-editor.org imho

IMHO, ideally we would write the docs in [Mallard](http://projectmallard.org/) format and add the help pages to the app itself.

This way, the help pages can be viewed with the normal help browser of the OS (e.g Yelp on GNOME, which you get by pressing `[F1]`), **and** an HTML export can be published online.

This is essentially what GNOME does, it works really well, see for example the Totem documentation:

* in the code: https://gitlab.gnome.org/GNOME/totem/tree/master/help
* online: https://help.gnome.org/users/totem/stable/

Docs can be translated, maintaining them in the code means they are kept up to date for a given version…

Also, the GNU IMP currently requires WebKitGtk, and AFAIK (although I haven't verified) it only needs it to display the help. Doing what I suggest essentially drops the dependency and [a bunch of code](https://github.com/glimpse-editor/Glimpse/tree/dev-g210/plug-ins/help-browser).

**Comment by TrechNex** [comment]
Indeed. For the purposes of 0.1.0 I think pointing at upstream docs is fine. We have a placeholder issue at #95 for replacing the online help.

I 100% support removing webkit as a dependency. It slows down our builds, makes the end result bigger than it needs to be & doesn't even work on some systems.

I'm not familiar with mallard. So far it has not cropped up in my 12 month career as a paid technical writer, so will be intrigued to find out more about it :)

**Comment by TrechNex** [comment]
Also linking #94 as removing webkit dependency in favour of a better help system impacts offline too

**Comment by TrechNex** [comment]
Moved to soft fork candidates so we can track progress after 0.1 releases.

**Comment by TrechNex** [comment]
We should determine if we already are pointing to online help from the flatpak or not. If so, we could close this in an earlier milestone.

**Comment by TrechNex** [comment]
Pushing out to 0.2.2 candidates
